sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeOperationResult.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Collection of <see cref="JobMetadata"/> objects corresponding
+    /// to a batch of documents, and information about the batch operation.
+    /// </summary>
+    public class AnalyzeOperationResult
+    {
+        /// <summary>
+        /// </summary>
+        /// <param name="entitiesResult"></param>
+        /// <param name="piiEntitiesResult"></param>
+        /// <param name="keyPhraseResult"></param>
+        /// <param name="map"></param>
+        internal AnalyzeOperationResult(EntitiesResult entitiesResult, PiiEntitiesResult piiEntitiesResult, KeyPhraseResult keyPhraseResult, IDictionary<string, int> map)
```

Comment:
could you add a more descriptive name for the `map` variable?
The more clear the name, the more readable the code is, so easier to maintain